_data/current.yml

```diff
@@ -191,6 +191,12 @@
   url: https://starwest.techwell.com/?utm_source=testingconferences
   twitter: techwell
   status: <a href="https://starwest.techwell.com/speak-starwest" target="_blank">CFP is Open</a> until February 12, 2023
+  
+- name: WeTest.Athens 2023
+  location: Athens, Greece, Europe
+  dates: "October 5-6, 2023"
+  url: https://www.linkedin.com/company/33295477/
```

Comment:
Is this the best url to use right now?

Reply:
Yes, we will update this, once the website is up.

Reply:
Site just came up : https://wetest-athens.gr/